### PR TITLE
Add display headings

### PR DESCRIPTION
**Pull request by kdelemme**

### CODE REVIEW

File: dist/css/bootstrap.css [hunk 1]
```diff
@@ -1442,6 +1442,26 @@ h6,
 .h6 {
   color: #829ca9;
 }
+.display-1 {
+  font-size: 6rem;
```
Comment (StevePole) [inline dist/css/bootstrap.css]: Bootstrap 4 uses rem but we don't, we should map these to pixel size.  Would suggest talking with Julian to come up with sizes.

Comment (StevePole) [reply dist/css/bootstrap.css]: All our line-heights should be a multiple of 8px, although font-size can be anything.

File: less/variables.less [hunk 2]
```diff
@@ -166,6 +165,20 @@
...
+
+@display1-size:       6rem;
+@display2-size:       5.5rem;
+@display3-size:       4.5rem;
+@display4-size:       3.5rem;
+
+@display1-weight:     300;
+@display2-weight:     300;
+@display3-weight:     300;
+@display4-weight:     300;
```
Comment (StevePole) [inline less/variables.less]: Talk to Julian about font-weight, I suspect we will match our existing headings.

**Review by StevePole — CHANGES_REQUESTED** [review]

File: dist/css/bootstrap.css (hunk 1 above)
Comment (kdelemme) [reply dist/css/bootstrap.css]: Thanks will talk with him

**Comment by StevePole** [comment]
Another thought, we use responsive typography.  So for all of our headings, they drop down to the next smallest size on mobile, the change happens at this breakpoint in type.less

@media (min-width: @screen-md-min) {

Could you add this?

File: dist/css/bootstrap.css [hunk 3]
```diff
@@ -1442,6 +1442,29 @@ h6,
...
+  margin-bottom: 8px;
+  font-size: 56px;
+  font-weight: bold;
+  line-height: 64px;
+}
+.display-2 {
+  margin-bottom: 4px;
+  font-size: 48px;
+  font-weight: bold;
+  line-height: 56px;
```
Comment (StevePole) [inline dist/css/bootstrap.css]: Line-height + margin should also be multiple of 8 if possible

**Review by StevePole — APPROVED** [review]

**Pull request merged**